index.js

```diff
@@ -17,6 +17,15 @@ const bot = controller.spawn({
   token: slackBotToken
 })
 
+const redisUrl = process.env.REDIS_URL;
+
+if (!redisUrl) {
```

Comment:
I made it optional 👍 
https://github.com/ohbarye/kpt-bot/pull/6/commits/88f1f8b941929631dde30b5f41740cb56898ac70